widgets/matplotlib/measurement/tofe_histogram.py

```diff
@@ -186,6 +186,9 @@ def on_draw(self):
                 QtWidgets.QMessageBox.Ok)
 
         colormap = cm.get_cmap(self.color_scheme.value)
+        
+        self.axes.set_ylim([y_min, y_max])
+        self.axes.set_xlim([x_min, x_max]) 
```

Comment:
If limits are set here, lines 202-205 no longer have any effect.
https://github.com/JYU-IBA/potku/blob/d70aa80f99162f3ad944c99b6a56e657e4ffb87f/widgets/matplotlib/measurement/tofe_histogram.py#L202-L205
These lines should be removed if there is no use for them.